### DISCUSSION TITLE
vultr japan Osaka support

### DISCUSSION THREAD

**Discussion by coolzoom** [VarunAgw/CloudPingTest#14] [Ideas] [answered]
![image](https://github.com/VarunAgw/CloudPingTest/assets/10232727/2cb0702b-a1d7-4a24-95c3-ef444b4f9fb0)

**Comment by VarunAgw** [comment, marked as answer]
Added

**Reply by coolzoom** [reply]
thanks!

**Comment by coolzoom** [comment]
I think the country needs modification, Osaka should belongs to Japan
![image](https://github.com/VarunAgw/CloudPingTest/assets/10232727/402dc9be-fea6-43ea-8c58-aadd484724f2)

**Comment by VarunAgw** [comment]
@coolzoom Thanks for your comment. I have fixed it :)

https://cloudpingtest.com/vultr
